baybe/utils/dataframe.py

```diff
@@ -455,8 +463,19 @@ def pretty_print_df(df: pd.DataFrame, max_rows: int = 6, max_columns: int = 4) -
         max_rows,
         "display.max_columns",
         max_columns,
+        "display.max_colwidth",
```

Comment:
out of curiosity: the description of this option on pandas tells me it does exactly what you do manually below, whats up here?

Reply:
This option only trims the *content* of columns, but not the header, see https://stackoverflow.com/questions/64976267/pandas-truncate-column-names.